### ISSUE TITLE
Support vite 3.2.4

### ISSUE THREAD

**Issue by TrendAlan** [Menci/vite-plugin-public-path#7]
Will you upate this plugin to support latest version 3.2.4?
Currently the peerDependencies is still '^2.8':

```
  "peerDependencies": {
    "vite": "^2.8"
  },
```

**Comment by Menci** [comment]
It works in Vite 3 in my project. Did you find any bug?

**Comment by TrendAlan** [comment]
When using npm version > v7, will report peerdependence error.
```
npm ERR! Found: vite@3.2.4
npm ERR! node_modules/vite
npm ERR!   dev vite@"^3.2.4" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer vite@"^2.8" from vite-plugin-public-path@0.0.3
npm ERR! node_modules/vite-plugin-public-path
npm ERR!   vite-plugin-public-path@"0.0.3" from the root project
```
Though I can use "npm config set legacy-peer-deps true" as workaround, maybe you just need to update the peerDependencies to support vite 3.x.x
 
Refer to: https://docs.npmjs.com/cli/v7/using-npm/config#legacy-peer-deps

**Comment by Menci** [comment]
Should be fixed in `v1.1.0`.
